### ISSUE TITLE
 Sample Apps should inform that you need to enter a payment method for your OpenAI API key to work

### ISSUE THREAD

**Issue by teresaqhoang** [microsoft/semantic-kernel#1755]
**Describe the bug**
Unable to generate bot response. Details: Error: 400: => Throttling: Too many requests, HTTP status: 429
when running Sample Apps with Open AI key

warn: Microsoft.SemanticKernel.IKernel[0]
Error executing action [attempt 1 of 1]. Reason: TooManyRequests. Will retry after 2000ms

**Expected behavior**
To solve the issue I hade to add my credit card in the: https://platform.openai.com/account/billing/payment-methods
After that is done it took some time before it work. I think about 10-15 minutes. The API payment is not included in the Premium, so if you pay for Premium, you must still pay for you API usage, separate, by adding a payment card.
After adding the card, remember to adjust your usage limits, so you have control: https://platform.openai.com/account/billing/limits

# **Ask**

Tutorials should inform that you need to enter a payment method when using Open AI for your API key to work.
In response to this issue: https://github.com/microsoft/semantic-kernel/issues/973

**Comment by Sailesh-2209** [comment]
@teresaqhoang I had a similar issue when I first added a payment method to OpenAI. And even after updating payment details, I had to wait almost a day to be able to run skills (joke skill) from the semantic kernel (there is a token limit for the first 24 hours maybe? not sure). I was getting a RateLimitError. There was also some weird behavior, where I was only getting this RateLimitError when using version 3.0.0 of semantic kernel and not with version 2.7.0. I stopped getting these errors about 24 hours after enabling billing on OpenAI Platform.
